flux-cli/src/test/resources/classifierUdf/options.txt

```diff
@@ -0,0 +1,10 @@
+--path "/Users/pbarber/Documents/GitHub/flux/flux-cli/src/test/resources/marklogic-docs"
```

Comment:
This looks like you didn't mean to include it?

Reply:
removed